### PR TITLE
Visualization per server room. Issue #1796

### PR DESCRIPTION
**Pull request by jszwaj**
Visualization grid is property of ServerRoom right now, not DataCenter. Connected to #1796

### CODE REVIEW

**Comment by mkurek** [comment]
Hi @jszwaj. Thanks for your contribution!

We have some thoughts about your PR:
- could you make datamigration to "copy" `visualization_cols_num` and `visualization_rows_num` to server room from it's data center?
- could you group server rooms by DC and "nest" them under DC label? Example:
  screen shot 2016-05-23 at 07 47 15
- please adjust demodata to new fields for server room (https://github.com/allegro/ralph/blob/ng/src/ralph/data_importer/management/commands/demodata.py)

**Comment by mkurek** [comment]
@jszwaj you forgot to replace `data_center.detail.rack` by `server_room.detail.rack` in `dc_view/static/partials/server_room/rack.html`, so clicking on rack (going to rack details) in SR view is not working. Could you fix it? Besides that, it's :ok: for me.

File: src/ralph/dc_view/templates/dc_view/server_room_view.html [hunk 1]
```diff
@@ -18,17 +18,25 @@
...
                       <li ui-sref-active="active">
-                        <a ui-sref="data_center.detail({dcId:{{ data_center.id }}})">
-                          {{ data_center }}
-                        </a>
+                        {{ data_center }}
+                        {% for server_room in data_center.server_rooms.all %}
+                          <ul>
+                            <li>
+                              <a ui-sref="server_room.detail({srId:{{ server_room.id }}})">
+                               {{ server_room }}
```
Comment (mkurek) [inline src/ralph/dc_view/templates/dc_view/server_room_view.html]: could you use `server_room.name` here?

**Comment by mkurek** [comment]
One more (i hope last) thing: now the list of DCs and SRs is pretty long. Could you decrease padding on this list to for example `rem-calc(3)` (look here: https://github.com/allegro/ralph/blob/ng/src/ralph/static/src/scss/components/visualization.scss#L12)?

**Comment by mkurek** [comment]
thanks @jszwaj ! :ok: for me :grinning:

**Comment by vi4m** [comment]
current screen please.

**Comment by jszwaj** [comment]
@vi4m  Screen of whole visualization view or only list of server rooms?

**Comment by mkurek** [comment]
@jszwaj the rest of the visualisation is unchanged - just the fragment with DCs and server rooms.

**Comment by jszwaj** [comment]
sr_list1

@vi4m , @mkurek is it ok?

**Comment by mkurek** [comment]
yes, it's ok for me.

**Comment by vi4m** [comment]
LGTM

**Pull request merged**